{{cookiecutter.project_slug}}/setup.py

```diff
@@ -45,6 +45,7 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
Oh we mention 3.11 here as well ... I'll let you decide how to harmonize declaration with what is really tested.